api/src/test/scala/hmda/api/http/InstitutionSpec.scala

```diff
@@ -0,0 +1,40 @@
+package hmda.api.http
+
+import java.io.File
+
+import akka.http.scaladsl.model.{ ContentTypes, HttpEntity, Multipart }
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import com.typesafe.config.ConfigFactory
+import hmda.api.RequestHeaderUtils
+import hmda.persistence.HmdaSupervisor
+import hmda.persistence.HmdaSupervisor.FindActorByName
+import hmda.persistence.demo.DemoData
+import hmda.persistence.institutions.InstitutionPersistence
+import org.iq80.leveldb.util.FileUtils
+import org.scalatest.{ BeforeAndAfterAll, MustMatchers, WordSpec }
+
+trait InstitutionSpec extends WordSpec with MustMatchers with BeforeAndAfterAll
```

Comment:
Thanks for the suggestion here and with renaming `InstitutionsHttpApiSpec` to something like `InstitutionsAuthSpec`, I was having trouble figuring out the naming and I think that this is right on!